Extensions/dnSpy.Analyzer/TreeNodes/ClassInterfaceSubtypedByNode.cs

```diff
@@ -0,0 +1,67 @@
+/*
+    Copyright (C) 2014-2024 mitch.capper+dns@gmail.com

```

Comment:
License header should just say `Copyright (C) 2024 mitch.capper+dns@gmail.com` here since this file got created this year and not in 2014 :p

Reply:
you don't know when I first wrote it, I take a loonggg time to put PR together;)